src/app/dashboard/users/page.tsx

```diff
@@ -44,7 +72,19 @@ async function getUsers(
     return
   }
 
-  const settings = getSettings()
+  const session = await getServerSession(authOptions)
```

Comment:
We can use the session from the parent. I would also read the `userId` there, which can then also be passed onto the media items for highlighting the current user.